workflows/funcscan.nf

```diff
@@ -156,16 +160,20 @@ workflow FUNCSCAN {
 
         if ( params.annotation_tool == "prodigal" ) {
             PRODIGAL_GFF ( ch_prepped_input, "gff" )
+            GUNZIP_FAA ( PRODIGAL_GFF.out.amino_acid_fasta )
+            GUNZIP_FNA ( PRODIGAL_GFF.out.nucleotide_fasta)
+            GUNZIP_GFF ( PRODIGAL_GFF.out.gene_annotations )
             ch_versions              = ch_versions.mix(PRODIGAL_GFF.out.versions)
-            ch_annotation_faa        = PRODIGAL_GFF.out.amino_acid_fasta
-            ch_annotation_fna        = PRODIGAL_GFF.out.nucleotide_fasta
-            ch_annotation_gff        = PRODIGAL_GFF.out.gene_annotations
+            ch_annotation_faa        = GUNZIP_FAA.out.gunzip
+            ch_annotation_fna        = GUNZIP_FNA.out.gunzip
+            ch_annotation_gff        = GUNZIP_GFF.out.gunzip
             ch_annotation_gbk        = Channel.empty() // Prodigal doesn't produce GBK
 
             if ( params.save_annotations == true ) {
                 PRODIGAL_GBK ( ch_prepped_input, "gbk" )
+                GUNZIP_GBK ( PRODIGAL_GBK.out.gene_annotations)
                 ch_versions              = ch_versions.mix(PRODIGAL_GBK.out.versions)
-                ch_annotation_gbk        = PRODIGAL_GBK.out.gene_annotations
+                ch_annotation_gbk        = GUNZIP_GBK.out.gunzip
```

Comment:
```suggestion
                ch_annotation_gbk        = PRODIGAL_GBK.out.gene_annotations # Prodigal GBK output stays zipped because it is currently not used by any downstream subworkflow.
```